apps/www/_blog/2023-12-15-v2-client-libraries.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: 'Supabase Libraries V2: Python, Swift, Kotlin, Flutter, and Typescript'
+description: 'Swift, Kotlin, C#, and Python are now stable and are onto the same version number v2.'
+launchweek: X
+categories:
+  - product
+tags:
+  - launch-week
+date: '2023-12-15'
+published_at: '2023-12-15:00:05.000-07:00'
+toc_depth: 3
+author: paul_copplestone
+image: lwx-client-libs/client-libs-OG.png
+thumb: lwx-client-libs/client-libs-thumb.png
+---
+
+Here is what you need to know:
+
+- [Swift](https://supabase.com/docs/reference/swift/introduction), [Kotlin](https://supabase.com/docs/reference/kotlin/introduction), [C#](https://supabase.com/docs/reference/csharp/introduction), and [Python](https://supabase.com/docs/reference/python/introduction) are now stable.
+- We’ve added even more support for mobile using React Native and [Expo](https://supabase.com/docs/guides/getting-started/tutorials/with-expo-react-native).
+- We’re consolidating all client libraries onto the same version number (v2) so that you get a consistent API across all libraries.
+
+## Version 2, for all libraries
+
+All of the libraries mentioned in this post are now on v2. This is because we want the API for each library to “move in lock step”. You should be able to jump around each of the client libraries and they should operate the same, barring any language idiosyncrasies.
+
+## Supabase Python v2
+
+Supabase Python is now stable thanks to the following maintainers: [Anand](https://github.com/anand2312), [Daniel Reinón García](https://github.com/dreinon), [Leynier Gutiérrez González](https://github.com/leynier), [Joel Lee](https://github.com/J0), and [Andrew Smith](https://github.com/silentworks/).
+
+Check out the [docs](https://supabase.com/docs/reference/python/introduction), as well as these Python examples to help you get started:
+
+- [Slack Consolidate: a slackbot built with Python and Supabase](https://supabase.com/blog/slack-consolidate-slackbot-to-consolidate-messages)
+- [Python data loading with Supabase](https://supabase.com/blog/loading-data-supabase-python)
+- [GitHub OAuth in your Python Flask app](https://supabase.com/blog/oauth2-login-python-flask-apps)
+
+<Admonition type="note" label="Vecs">
+
+Don’t forget that we also have Vecs, a Python client for managing and querying vector stores in Postgres with the pgvector extension. [Get started](https://supabase.com/docs/guides/ai/python/api)!
+
+</Admonition>
```

Comment:
For some reason this doesn't look like it does in wednesday's article?

https://supabase.com/blog/supabase-branching#data-seeding

Reply:
<img width="728" alt="image" src="https://github.com/supabase/supabase/assets/10214025/cd01f59d-8ff3-4bac-8392-913da39fcb06">
<img width="803" alt="image" src="https://github.com/supabase/supabase/assets/10214025/51e50a80-190a-470e-901e-42957355c95c">